src/lxc/execute.c

```diff
@@ -111,16 +111,15 @@ static struct lxc_operations execute_start_ops = {
 };
 
 int lxc_execute(const char *name, char *const argv[], int quiet,
```

Comment:
Also, I've switched `lxc-execute` to rely on public API. A full switch to API-only will happen when we exposed a log function in our API. A PR for this already exists.